package.json

```diff
@@ -58,6 +58,7 @@
     "file-loader": "1.1.11",
     "html-webpack-plugin": "3.2.0",
     "jest": "23.1.0",
+    "redux-mock-store": "^1.5.3",
```

Comment:
if you guys prefer to lock versions, I will. However, my preference is typically to let that be the job of the lockfile & use `yarn --pure-lockfile` if you dont want to pull in updates.

Reply:
That's ok with me if we want to move to that, but the README.md needs to be updated. Travis CI just uses `yarn install`, so that would need to be updated as well.